### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by pushkar2112**
Fix typos in the documentation and edit some formatting.

### CODE REVIEW

**Review by 7yl4r — APPROVED** [review]

**Comment by ocefpaf** [comment]
@pushkar2112 can you fix the conflict we are seeing here?

**Comment by pushkar2112** [comment]
> @pushkar2112 can you fix the conflict we are seeing here?

 @ocefpaf I will look into it and try to resolve the conflict!

File: README.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ dev version
     pip install git+git://github.com/sckott/pyobis.git#egg=pyobis
 
 
-library API
+Library APIs
```
Comment (ocefpaf) [inline README.rst:31]: I believe that within the context of a single library API is OK.

```suggestion
Library API
```

File: README.rst [hunk 2]
```diff
@@ -104,7 +109,7 @@ Checklist module
 Meta
 ====
 
-* License: MIT, see `LICENSE file <LICENSE>`__
+* License: MIT,  `see here <LICENSE>`__
```
Comment (ocefpaf) [inline README.rst:112]: I prefer to be explicit, that is the name of the file and the link to it. 

```suggestion
* License: MIT,  `see LICENSE file <LICENSE>`__
```

**Comment by ocefpaf** [comment]
The tests are failing but not b/c of this PR. Let's merge this. Thanks @pushkar2112!

File: README.rst (hunk 2 above)
Comment (pushkar2112) [reply README.rst]: Noted!

**Pull request merged**